framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -330,6 +333,106 @@ public void processClassTree(ClassTree classTree) {
         super.visitClass(classTree, null);
     }
 
+    /**
+     * Check that the field invariant meets the following requirements:
```

Comment:
OK, let's keep thinking about good terminology, and reviewing all the documentation at once will help us with that.  Thanks.